apps/sertra/api/PackageDataFetcher.ts

```diff
@@ -13,35 +13,27 @@ export const FetchListingData = async (listingId: string) => {
     x.json()
   );
   const listing = await listing_data_fetch;
-  const mods = await mod_data_fetch;
+  const mods = (await mod_data_fetch) as TSListingMod[];
 
   const all_versions = new Map();
-  mods.map((mod: TSListingMod) => {
-    mod.versions.map((version) => {
+  for (const mod of mods as Array<TSListingMod>) {
```

Comment:
I believe there doesn't have to be any type casts here (`as X` statements). line 16 already takes care of the typing